examples/sas_logical_interconnect_group.rb

```diff
@@ -15,23 +15,24 @@
   password: ENV['ONEVIEWSDK_PASSWORD']
 }
 
-# This resource is only available for API300::Synergy, so we need to set these attributes
-# to ensure it loads the correct resource_provider module. You can also set the api_version
-# and api_variant properties on each resource definition below (see the README).
+# This resource is only available for Synergy and API 300 and onwards, so we need
```

Comment:
This resource is only available for Synergy *on* API 300 onwards, and these attributes *must* be set to ensure the correct resource_provider module is loaded.
The api_version and api_variant properties can also be set on each resource definition *bellow* (see the README)